### DISCUSSION TITLE
Getting the progress while counting down

### DISCUSSION THREAD

**Discussion by PlacidSerene** [ndresx/react-countdown#216] [Q&A] [answered]
Is there a way to get the percentage complete for the countdown. I'm trying to make a circular progress bar to display the progress when counting down. Thank you!

**Comment by ndresx** [comment, marked as answer]
The only way I could think of right now is to use this object returned in the renderer or other callback functions: https://github.com/ndresx/react-countdown#renderer. There's a `total` value that basically represents the time left in milliseconds. Since you know the initial time, you could use one of the callbacks or https://github.com/ndresx/react-countdown#calctimedeltadate-options to calculate the initial `total` time left, and then do the rest of the math by calculating the percentage between this initial `total` and the current `total`. Does this help?

**Reply by PlacidSerene** [reply]
I think it could work, let me try it and I'll comeback later to let you know :) Thank you Martin!

**Reply by PlacidSerene** [reply]
Works like a charm! I have included the codesanbox below just in case anyone come up with the same question. Thank you!
https://codesandbox.io/s/pedantic-https-fobzsg?file=/src/App.js

**Reply by ndresx** [reply]
Cool! 👏
